membership-attribute-service/app/prodtest/VariablesUpdatedOnSchedule.scala

```diff
@@ -0,0 +1,33 @@
+package prodtest
+
+import akka.actor.ActorSystem
+import com.gu.memsub.util.ScheduledTask
+import com.typesafe.scalalogging.LazyLogging
+import services.ScanamoFeatureToggleService
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+
+class VariablesUpdatedOnSchedule(featureToggleService: ScanamoFeatureToggleService)(implicit ec: ExecutionContext, system: ActorSystem) extends LazyLogging {
+
+  private var percentageTrafficForZuoraLookup = 0
```

Comment:
It just wasn't obvious to me how to avoid a mutable variable once I needed a globally accessible value that changes on a schedule. 😅